modules/get-started/pages/release-notes/helm-charts.adoc

```diff
@@ -36,13 +36,25 @@ An updated versioning scheme now better supports and tracks compatible Redpanda
 
 See also: xref:upgrade:k-compatibility.adoc[].
 
-=== Connectors subchart removed
+=== Kafka Connect integration removed from Helm chart
 
-The connectors subchart has been removed from the Helm chart because it is not officially supported.
+Starting with version 25.1.x of the Redpanda Helm chart, Kafka Connect is no longer bundled as a subchart.
 
-:tip-caption: Try Redpanda Connect
+If you require Kafka Connect:
 
-TIP: For a supported and scalable Kafka Connect alternative, consider using Redpanda Connect. For more information, see xref:redpanda-connect:get-started:index.adoc[].
+- You must install the `connectors` Helm chart separately after deploying Redpanda.
+- This change affects both Helm and Operator-based Redpanda deployments on Kubernetes.
```

Comment:
I don't really understand this list. Instead of a list, should it be 1 sentence: "If you require Kafka Connect, you must install the..." And maybe the 2nd bullet goes up above? Is there a different requirement for the Operator?